tests/assets/ansible/roles/caasp/tasks/Suse.yaml

```diff
@@ -1,10 +1,12 @@
 ---
 - name: add repositories
   vars:
+    sle_major: "{{ ansible_distribution_version.split('.')[0] }}"
+    sle_minor: "{{ ansible_distribution_version.split('.')[1] }}"
     repositories:
-      caasp_devel: http://download.suse.de/ibs/Devel:/CaaSP:/4.0/SLE_15_SP1/
-      suse_ca: http://download.suse.de/ibs/SUSE:/CA/SLE_15_SP1/
-      storage: http://download.suse.de/ibs/SUSE:/SLE-15-SP2:/Update:/Products:/SES7/standard
+      caasp_devel: http://download.suse.de/ibs/Devel:/CaaSP:/4.0/SLE_{{ sle_major }}_SP{{ sle_minor }}/
```

Comment:
wrongly rebased I guess

Reply:
The file looks good to me. What is wrong ?

Reply:
True. all good. sorry for the noise.

Reply:
With https://github.com/SUSE/rookcheck/pull/107 , I think this change is wrong, isn't it?

Reply:
I forgot to push my rebase I think.
repo are hardcoded for now to caasp 4